### PR TITLE
Add the missing regions

### PR DESCRIPTION
**Pull request by jocgir**
I sorted the regions as I got them with their name from powershell command Get-AWSRegion.

### CODE REVIEW

**Comment by ekalinin** [comment]
Hey!

Thanks for tha patch! Please, fix tests.

File: aws_test.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func TestAWSRegionCheckLatencyHTTP(t *testing.T) {
 	got := regions[0].GetLatency()
 	want := 15.0
 
-	if got < want || got > want+2 {
```
Comment (jocgir) [inline aws_test.go:67]: I am not sure why you put +2 here. I ran the test 1000 times and got errors 5 out of 1000 times. I just extended the delay and all tests are working.

File: aws_test.go [hunk 2]
```diff
@@ -189,7 +189,7 @@ func TestAWSRegionsSetService(t *testing.T) {
 	regions.SetService(service)
 
 	if regions[0].Service != service || regions[len(regions)-1].Service != service {
-		t.Errorf("failed: not setted, regions=%q, service=%s", regions, service)
```
Comment (jocgir) [inline aws_test.go:192]: setted is not supposed to be a valid word 😉

**Comment by jocgir** [comment]
@ekalinin  Hello, I fixed the tests. Do you need something else?

**Comment by ekalinin** [comment]
Thanks!

**Pull request merged**